backend/app/services/project_service.py

```
@@ -0,0 +1,476 @@
+        project_id: Project ID
+        user_id: User ID (for ownership verification)
+
+    Raises:
+        HTTPException: If project not found
+    """
+    project = (
+        db.query(Project)
+        .filter(
+            Project.id == project_id,
+            Project.user_id == user_id,
+            Project.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)